README.md

```diff
@@ -22,3 +22,7 @@ Every deployment includes updates to continuous monitoring.
 > __Compliance as Code.__
 
 The full website is online at [http://opencontrol.xyz](http://opencontrol.xyz).
```

Comment:
Maybe we should be promoting the http://open-control.org url instead - I've had push-back from some agencies on the .xyz TLD.